### PR TITLE
bpo-37698: Update doc of PyBuffer_ToContiguous

### PR DESCRIPTION
**Pull request by shihai1991**
https://bugs.python.org/issue37698

Automerge-Triggered-By: @matrixise

### CODE REVIEW

**Review by mangrisano — APPROVED** [review]
LGTM. Thank you for opening this PR.

**Comment by shihai1991** [comment]
> LGTM. Thank you for opening this PR.

thanks for review :)

**Comment by shihai1991** [comment]
cc @pitrou

File: Doc/c-api/buffer.rst [hunk 1]
```diff
@@ -478,7 +478,7 @@ Buffer-related functions
 .. c:function:: int PyBuffer_ToContiguous(void *buf, Py_buffer *src, Py_ssize_t len, char order)
 
    Copy *len* bytes from *src* to its contiguous representation in *buf*.
-   *order* can be ``'C'`` or ``'F'`` (for C-style or Fortran-style ordering).
+   *order* can be ``'C'`` or ``'F'`` or ``'A'`` (for C-style or Fortran-style ordering or either one).
```
Comment (matrixise) [inline Doc/c-api/buffer.rst]:
```
ordering or either one).
```
should be on the next line, could you rewrap your sentence?

Thank you.

**Review by matrixise — CHANGES_REQUESTED** [review]
Thank you for your contribution, but could you read my comment and just update your PR?

Thank you

File: Doc/c-api/buffer.rst (hunk 1 above)
Comment (shihai1991) [reply Doc/c-api/buffer.rst]: Thanks, Stéphane. It is done.

**Comment by shihai1991** [comment]
I have made the requested changes; please review again

**Review by matrixise — APPROVED** [review]

**Pull request merged**